gem/Rakefile

```diff
@@ -50,12 +71,14 @@ platform_config = {
 def copy_files(cnf, meterpreter_dest)
   cnf[:sources].each do |f|
     cnf[:extensions].each do |ext|
-      Dir.glob("#{f}/*.#{ext}").each do |bin|
-        target = File.join(meterpreter_dest, File.basename(bin))
+      Dir.glob("#{f}/**/*.#{ext}").each do |bin|
+        target = File.join(meterpreter_dest, bin.sub(f, ''))
```

Comment:
Is this the same as `::Pathname.relative_path_from(f, bin)` - or the inverse `::Pathname.relative_path_from(bin, f)` 😄